test/test_sparse.py

```diff
@@ -1283,6 +1283,25 @@ def fn(S, D1, D2, beta=beta, alpha=alpha):
         test_shape(7, 8, 9, 20, False, (1, 1))
         test_shape(7, 8, 9, 20, True, (1, 1))
 
+    @onlyCPU
+    @coalescedonoff
+    def test_sparse_addmm_bfloat16(self, device, coalesced):
```

Comment:
What stands in the way of extending above test test_sparse_addmm by adding bfloat16 to the dtypes decorator?

Reply:
I have tried to add UT by this way, but the test brings in five more unsupported OPs.

Reply:
Would be equally as easy to add bfloat16 support for those as well as in this case?

Reply:
Yes, but you could change that precision for bfloat16 specifically. You can branch on the dtype input and set a different precision for comparison.